client/css/style.css

```diff
@@ -860,6 +860,10 @@ button,
 	color: #f00;
 }
 
+#chat .msg.toggle .time {
+	visibility: hidden;
+}
+
```

Comment:
Not sending a timestamp requires more work than necessary.

Using `display` will break the layout as we use tables for messages.